openlibrary/core/lending.py

```diff
@@ -1103,9 +1103,9 @@ def _post(self, **payload):
             ).json()
             logger.info("POST response: %s", jsontext)
             return jsontext
-        except Exception:  # TODO: Narrow exception scope
+        except (JSONDecodeError, Exception):  # TODO: Narrow exception scope
```

Comment:
We should split this into **just** `JSONDecodeError` and/or 403... And any other `Exception` and continue to raise

Reply:
We might also consider using `timeout=1` or something to reduce the wait time for calls we know should be fast.